src/bin/report_collector.rs

```diff
@@ -308,6 +337,10 @@ async fn ipa(
         write!(file, "{}", serde_json::to_string_pretty(&actual)?)?;
     }
 
+    tracing::info!("{m:?}", m = ipa_query_config);
```

Comment:
still needed?

Reply:
Yes this is a debug message which helps us know what is being run.